x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/crawl_details_flyout/crawl_details_preview.tsx

```diff
@@ -25,6 +30,14 @@ export const CrawlDetailsPreview: React.FC = () => {
 
   return (
     <>
+      <CrawlDetailsSummary
+        crawlerLogsEnabled={crawlerLogsEnabled}
+        crawlType={crawlRequest.type}
+        domainCount={crawlRequest.crawlConfig.domainAllowlist.length}
+        crawlDepth={crawlRequest.crawlConfig.maxCrawlDepth}
+        stats={crawlRequest.stats || null}
```

Comment:
Added additional code coverage: https://github.com/elastic/kibana/commit/a0b37052919355a95c5cd53a5167ebbf4a71ed37

Reply:
You can cherry pick that commit. Also address a formatting issue.

Reply:
Did this for you to keep things moving.